security/tailscale/src/opnsense/mvc/app/controllers/OPNsense/Tailscale/forms/settings.xml

```diff
@@ -29,4 +29,11 @@
         <type>checkbox</type>
         <help>Accept subnet routes that other nodes advertise.</help>
     </field>
+    <field>
+        <id>settings.loginTimeout</id>
+        <label>Login timeout</label>
+        <type>text</type>
+        <help>Maximum time to wait for successful login, in seconds.  Set to 0 to wait indefinitely, however this may prevent
```

Comment:
Typewriter end of sentence is cool, but the same excess whitespace rule applies regarding translations.